plugins/automation/admin/assets/javascripts/admin/components/automation-list.gjs

```diff
@@ -0,0 +1,242 @@
+import Component from "@glimmer/component";
+import { fn } from "@ember/helper";
+import { on } from "@ember/modifier";
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import DButton from "discourse/components/d-button";
+import DPageSubheader from "discourse/components/d-page-subheader";
+import DToggleSwitch from "discourse/components/d-toggle-switch";
+import avatar from "discourse/helpers/avatar";
+import formatDate from "discourse/helpers/format-date";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { escapeExpression } from "discourse/lib/utilities";
+import { i18n } from "discourse-i18n";
+import AdminConfigAreaEmptyList from "admin/components/admin-config-area-empty-list";
+
+const THRESHOLD = 10;
```

Comment:
Maybe can add some descriptive part here to explain what it's a threshold for? 🙂

Reply:
good point fixed!